### PR TITLE
fixing documentation link and adding groq package

### PR DESCRIPTION
**Pull request by viraatdas**
Simple link fix in readme and adding groq to dependencies

----

| :rocket: | This description was created by [Ellipsis](https://www.ellipsis.dev) for commit a58e55015e14f10704aab00a6190cc5f6b9333b5  | 
|--------|--------|

### Summary:
Updated a documentation link in `README.md` to point to `https://help.getzep.com/concepts`.

**Key points**:
- Updated documentation link in `README.md`.
- Changed link from `https://help.getzep.com/Graphiti/Graphiti` to `https://help.getzep.com/concepts`.

----
Generated with :heart: by [ellipsis.dev](https://www.ellipsis.dev)

### CODE REVIEW

**Comment by viraatdas** [comment]
I have read the CLA Document and I hereby sign the CLA

File: pyproject.toml [hunk 1]
```diff
@@ -23,6 +23,7 @@ sentence-transformers = "^3.0.1"
 diskcache = "^5.6.3"
 openai = "^1.38.0"
 tenacity = "<9.0.0"
+groq="^0.11.0"
```
Comment (paul-paliychuk) [inline pyproject.toml:26]: We intentionally didn't add groq and anthropic as direct dependencies in poetry as it is an expectation on the user's end to have those installed in their environment if they are using Grok or Anthropic LLM clients. We don't want to have all the users install those.

**Review by paul-paliychuk — CHANGES_REQUESTED** [review]
@viraatdas Hi, thanks for opening the PR. The documentation link is def broken (oops), but can you please revert the grok dep addition?

**Comment by paul-paliychuk** [comment]
Fixed the doc link in #84 , will be closing this PR

**Pull request closed without merging**